pkg/convert/convert.go

```diff
@@ -66,51 +72,17 @@ func ResourcesToYamlMachineTypes(resources []map[string]interface{}, outputFile
 		panic(err)
 	}
 	for _, resource := range resources {
-		if resource["resource_type"] == "compute.node" {
+		if resource[ResourceType] == "compute.node" {
 			// Create block for this machine type
 			id := helper.AssignStringValue(resource["id"])
 			machineTypeString := []byte("    - id: " + id +
-				"\n      name: " + resource["name"].(string) + "\n      cloud_provider_id: " +
-				resource["cloud_provider_id"].(string) + "\n      cpu_cores: " +
-				resource["cpu_cores"].(string) + "\n      memory: " + resource["memory"].(string) +
+				"\n      name: " + resource[Name].(string) + "\n      cloud_provider_id: " +
+				resource[CloudProviderId].(string) + "\n      cpu_cores: " +
+				resource[CpuCores].(string) + "\n      memory: " + resource[Memory].(string) +
 				"\n      category: " +
-				resource["category"].(string) + "\n      size: " + resource["size_pretty"].(string) +
+				resource[Category].(string) + "\n      size: " + resource[SizePretty].(string) +
 				"\n      generic_name: " +
-				resource["generic_name"].(string) + "\n")
```

Comment:
I think that formatting this code differently would improve readability
```go
			machineTypeString := []byte("    - id: " + id +
				"\n      name: " + resource[Name].(string) +
				"\n      cloud_provider_id: " + resource[CloudProviderId].(string) +
				"\n      cpu_cores: " + resource[CpuCores].(string) +
				"\n      memory: " + resource[Memory].(string) +
				"\n      category: " + resource[Category].(string) +
				"\n      size: " + resource[SizePretty].(string) +
				"\n      generic_name: " + resource[GenericName].(string) + "\n")
```

Another option: instead of string concatenation, you could use `fmt.Sprintf`
```go
			machineTypeString = []byte(fmt.Sprintf(
				"    - id: %s\n"+
					"      name: %s\n"+
					"      cloud_provider_id: %s\n"+
					"      cpu_cores: %s\n"+
					"      memory: %s\n"+
					"      category: %s\n"+
					"      size: %s\n"+
					"      generic_name: %s\n",
				id, resource[Name].(string), resource[CloudProviderId].(string), resource[CpuCores].(string),
				resource[Memory].(string), resource[Category].(string), resource[SizePretty].(string),
				resource[GenericName].(string)))
```